### ISSUE TITLE
missing image alt text

### ISSUE THREAD

**Issue by bashbaug** [KhronosGroup/OpenCL-Docs#995]
Another issue identified by https://pagespeed.web.dev/ is that our spec images do not have "alt" text.  Having alt text will make our specs more accessible for screen readers.  Asciidoctor supports images with alt text:

https://docs.asciidoctor.org/asciidoc/latest/macros/images/

**Comment by bashbaug** [comment]
I actually can't find any images without "alt" text, and another checker I ran didn't find any images with missing alt text, so I think I'm just going to close this issue.
